setup.sh

```diff
@@ -108,26 +131,27 @@ function confirmChoices() {
 
 function installAnew() {
     local projects
-    projects=($*)
-    for project in $projects; do
+    projects=("$@")
+    for project in "${projects[@]}" ; do
         (
-            cd $wd &&
-            git clone --quiet git://github.com/dlang/$project.git &&
-            touch $tempdir/$project
+        git clone "git://github.com/$githubUser/$project.git" "$wd/$project"
```

Comment:
`git://` is insecure without an external source of validation, like a known commit SHA1, submodule, or signed tag.